mongodb/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM gitpod/workspace-full:latest
+
+# Install mongodb
+RUN sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 9DA31620334BD75D9DCB49F368818C72E52529D4
+RUN echo "deb [ arch=amd64 ] https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/4.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-4.0.list
+RUN sudo apt-get update \
+ && sudo apt-get install -y mongodb-org \
+ && sudo apt-get clean \
+ && sudo rm -rf /var/cache/apt/* /var/lib/apt/lists/* /tmp/*
+RUN sudo mkdir -p /data/db
+RUN sudo chown gitpod:gitpod -R /data/db

```

Comment:
Note: Every `RUN` command will create a new Docker layer. I don't think we need to create so many layers here.

Could you please briefly rework these lines from:

```Dockerfile
RUN sudo mkdir -p /data/db
RUN sudo chown gitpod:gitpod -R /data/db
```

to:

```Dockerfile
RUN sudo mkdir -p /data/db \
 && sudo chown gitpod:gitpod -R /data/db
```

please?

Reply:
Done